### ISSUE TITLE
how can i use this

### ISSUE THREAD

**Issue by CraftyBambi** [ox-y/chrome-extension-roblox-cookie-logger#56]
look why not theres bunch of tutorials of cookie logging in yt

**Comment by bestadamdagoat** [comment]
yeah just follow his tutorial [here](https://shareit.pics/19BN7W.txt)

**Comment by bestadamdagoat** [comment]
![image](https://github.com/tcpcon/chrome-extension-logger-poc/assets/66372881/d4f74393-dbb1-42be-bba5-2a060d7092a3)

**Comment by CraftyBambi** [comment]
rude wtf did i do

**Comment by bestadamdagoat** [comment]
survival of the fittest lil bro. don't go into the beaming scene expecting not to get beamed.

**Comment by dem1se** [comment]
what did he do? I'm scared to click the embedded link

**Comment by dem1se** [comment]
nvm it loaded the image loaded

**Comment by Procoder549** [comment]
> ![image](https://user-images.githubusercontent.com/66372881/258893737-d4f74393-dbb1-42be-bba5-2a060d7092a3.png)

lol i use malwarebytes extension
